pkg/services/sqlstore/sqlstore.go

```diff
@@ -236,7 +236,8 @@ func (ss *SQLStore) buildConnectionString() (string, error) {
 		}
 
 		if isolation := ss.dbCfg.IsolationLevel; isolation != "" {
-			cnnstr += "&tx_isolation=" + isolation
+			val := url.QueryEscape(fmt.Sprintf("'%s'", isolation))
+			cnnstr += fmt.Sprintf("&tx_isolation=%s", val)
```

Comment:
Also looks like they decided to rename it to `transaction_isolation`: 

> transaction_isolation was added in MySQL 5.7.20 as a synonym for tx_isolation, which is now deprecated and is removed in MySQL 8.0. Applications should be adjusted to use transaction_isolation in preference to tx_isolation. See the description of transaction_isolation for details.

(from https://dev.mysql.com/doc/refman/5.7/en/server-system-variables.html#sysvar_transaction_isolation)

Reply:
Yes, that's a later problem. See https://github.com/grafana/grafana/issues/31371#issuecomment-877070417

Reply:
Hello
```
Unknown system variable 'tx_isolation'
```
still persists for MySQL 8.0.x
is there any possibility to configure transaction_isolation instead of tx_isolation?